apps/course-authoring/scripts/pipeline/notarize.js

```diff
@@ -1,7 +1,7 @@
-import { notarize } from 'electron-notarize';
-import { build } from '../../package.json';
+const { notarize } = require('electron-notarize');
```

Comment:
I've got TS complaining at me about these changes:

![image](https://user-images.githubusercontent.com/98419841/164736938-a861e554-9223-4a8b-a536-324ebc83bc71.png)

If it has to be in commonJS we can disable this rules for this file at least adding `/* eslint-disable @typescript-eslint/no-var-requires */` to the top. 

We also have the option to disable it globally, it will allow us to use `require()` anywhere. What do you guys think we should go for?

Reply:
I am not getting that. I did restart VS Code as well... strange, it makes sense for it to complain